exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -17,17 +17,56 @@
   <a href="/queries">back</a><br/>
   <div class="page-header">
   </div>
-  <div class="table-responsive">
-    <table class="table table-hover">
-      <tbody>
-        <#list model as stat>
-          <tr>
-            <td style="border:none;"><b>${stat.getName()}</b></td>
-            <td style="border:none; font-family: Courier;">${stat.getValue()}</td>
-          </tr>
-        </#list>
-      </tbody>
-    </table>
+
+  <#list model.getDrillbits() as drillbit>
+    <#if !drillbit.isVersionMatch()>
+      <div id="message" class="alert alert-danger alert-dismissable">
+        <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+        <strong>Drillbits in the cluster have different versions.</strong>
+      </div>
+      <#break>
+    </#if>
+  </#list>
+
+  <div class="row">
+    <div class="col-md-6">
+      <h3>General Info</h3>
+      <div class="table-responsive">
+        <table class="table table-hover">
+          <tbody>
+            <#assign props = model.getProps()>
+            <#list props?keys as key>
+              <tr>
+                <td style="border:none;"><b>${key}</b></td>
+                <td style="border:none; font-family: Courier;">${props[key]}</td>
```

Comment:
Styles should be pulled out into the style sheet. Very hard to maintain inline styles. And, why do we need to use a different font?

This is web UI, so it is not ensured that every browser has Courier. Instead, standard practice is to list multiple fonts, with a fallback to, say, monospace. See [this page](http://www.w3schools.com/cssref/css_websafe_fonts.asp).